plaid/item.go

```diff
@@ -3,15 +3,33 @@ package plaid
 import (
 	"encoding/json"
 	"errors"
+	"time"
 )
 
 type Item struct {
-	AvailableProducts []string `json:"available_products"`
-	BilledProducts    []string `json:"billed_products"`
-	Error             Error    `json:"error"`
-	InstitutionID     string   `json:"institution_id"`
-	ItemID            string   `json:"item_id"`
-	Webhook           string   `json:"webhook"`
+	AvailableProducts []string   `json:"available_products"`
+	BilledProducts    []string   `json:"billed_products"`
+	Error             Error      `json:"error"`
+	InstitutionID     string     `json:"institution_id"`
+	ItemID            string     `json:"item_id"`
+	Webhook           string     `json:"webhook"`
+	Status            ItemStatus `json:"status"`
+}
+
+type ItemStatus struct {
+	Transactions ProductStatus `json:"transactions,omitempty"`
+	Investments  ProductStatus `json:"investments,omitempty"`
+	LastWebhook  WebhookStatus `json:"last_webhook,omitempty"`
+}
+
+type ProductStatus struct {
+	LastFailedUpdate     time.Time `json:"last_failed_update,omitempty"`
```

Comment:
Should `time.Time` be `*time.Time` since these timestamp fields are optional?

Reply:
Good question, @skansal-plaid. Since the zero value for `time.Time` is invalid, we don't need to distinguish between the zero value of `time.Time` and the zero value of `*time.Time`. The `webhook` field is also nullable, but we use `string` instead of `*string` for the same reason.

Reply:
@jacksiegel Thanks, how about the fields contained in `ItemStatus` all of which are also optional?